### ISSUE TITLE
Where is mitchellh.vmware coming from?

### ISSUE THREAD

**Issue by kelseyhightower** [mitchellh/packer#2130]
When building the following packer configuration template (see below), I get the following output:

## Output

```
==> vmware-vmx: Gracefully halting virtual machine...
    vmware-vmx: Waiting for VMware to clean up after itself...
==> vmware-vmx: Deleting unnecessary VMware files...
    vmware-vmx: Deleting: output-vmware-vmx/564d82f7-6272-b85b-87ee-c4c7952a2362.vmem
    vmware-vmx: Deleting: output-vmware-vmx/kubestack-solo.plist
    vmware-vmx: Deleting: output-vmware-vmx/vmware.log
==> vmware-vmx: Cleaning VMX prior to finishing up...
    vmware-vmx: Unmounting floppy from VMX...
    vmware-vmx: Detaching ISO from CD-ROM device...
==> vmware-vmx: Compacting the disk image
==> vmware-vmx: Running post-processor: compress
==> vmware-vmx (compress): Creating archive for 'mitchellh.vmware'
Build 'vmware-vmx' finished.

==> Builds finished. The artifacts of successful builds are:
--> vmware-vmx: 'mitchellh.vmware' compressing: kubestack-solo-vmware.tar.gz
```

I'm not user where `mitchellh.vmware` is coming from.

## Packer config:

```
cat kubestack-solo.json 
{
  "variables": {
    "kubestack_file": "{{env `KUBESTACK_FILE`}}",
    "source_path": "{{env `SOURCE_PATH`}}",
    "ssh_username": "{{env `SSH_USERNAME`}}",
    "ssh_key_path": "{{env `SSH_KEY_PATH`}}"
  },
  "builders": [{
    "type": "vmware-vmx",
    "headless": true,
    "source_path": "{{user `source_path`}}",
    "ssh_username": "{{user `ssh_username`}}",
    "ssh_key_path": "{{user `ssh_key_path`}}",
    "shutdown_command": "sudo poweroff",
    "vm_name": "kubestack-solo"
  }],
  "provisioners": [{
        "type": "file",
        "source": "static",
        "destination": "/tmp"
    },
    {
        "type": "shell",
        "inline": [
            "sudo mkdir -p /etc/kubernetes",
            "sudo mkdir -p /opt/bin",
            "sudo tar -xvf /tmp/static/{{user `kubestack_file`}} -C /opt/bin",
            "sudo chmod +x -R /opt/bin",
            "sudo chown root:root -R /opt/bin",
            "sudo mv /tmp/static/*.service /etc/systemd/system",
            "sudo mv /tmp/static/kubernetes.env /etc/kubernetes.env",
            "sudo mv /tmp/static/node.json /etc/kubernetes/node.json",
            "sudo systemctl enable etcd",
            "sudo systemctl enable kube-apiserver",
            "sudo systemctl enable kube-controller-manager",
            "sudo systemctl enable kube-scheduler",
            "sudo systemctl enable docker",
            "sudo systemctl enable kube-kubelet",
            "sudo systemctl enable kube-proxy",
            "rm -rf /tmp/static"
        ]
    }],
    "post-processors": [{
        "type": "compress",
        "output": "kubestack-solo-vmware.tar.gz"
    }]
}
```

**Comment by okeefm** [comment]
Looks like it's coming from https://github.com/mitchellh/packer/blob/master/builder/vmware/common/artifact.go#L12, which sets the `artifact.BuilderId` to `mitchellh.vmware`. Right now it's a constant; should it be set instead to the declared VM name in the JSON?

**Comment by mitchellh** [comment]
I just wanted you see my name @kelseyhightower.

Uh, I think the output here might just be weird. The builder ID shouldn't be used in this way. Will be fixed by #2119
